### ISSUE TITLE
Gatsby website fails to load

### ISSUE THREAD

**Issue by sedubois** [gatsbyjs/gatsby#9939]
## Description

As I was reading the latest Gatsby gazette, I clicked [this link](https://www.gatsbyjs.org/blog/2018-11-07-gatsby-for-apps/) to the "Gatsby for apps" blog post, but the page remained white. It loaded after reopening it again.

It looks like a similar error as I [reported previously](https://github.com/gatsbyjs/gatsby/issues/3582), but Kyle said that one should have been fixed in v2.

### Steps to reproduce

Don't know how to reproduce.

### Expected result

The page should have loaded.

### Actual result

Page was entirely blank. Console showed:

```
Unhandled promise rejection Error: "Loading chunk 43 failed.
(error: https://www.gatsbyjs.org/43-6892247c8fc728767d9f.js)"
	n https://www.gatsbyjs.org/webpack-runtime-f4da6aa995ca964afdff.js:1:6825

L/</t< https://www.gatsbyjs.org/app-e85d14a7fe2197fb94f6.js:1:31219
e.exports https://www.gatsbyjs.org/app-e85d14a7fe2197fb94f6.js:1:69811
L/< https://www.gatsbyjs.org/app-e85d14a7fe2197fb94f6.js:1:31096
e.exports https://www.gatsbyjs.org/app-e85d14a7fe2197fb94f6.js:1:67993
<anonymous> https://www.gatsbyjs.org/app-e85d14a7fe2197fb94f6.js:1:45186
y https://www.gatsbyjs.org/app-e85d14a7fe2197fb94f6.js:1:45044
b https://www.gatsbyjs.org/app-e85d14a7fe2197fb94f6.js:1:45064
```

<img width="1273" alt="screenshot 2018-11-15 at 10 27 28" src="https://user-images.githubusercontent.com/4217871/48540330-7c35cc00-e8c2-11e8-9281-b75bec90d581.png">

### Environment

Firefox 63.0.1 (64-bit), macOS 10.14.1.

**Comment by kakadiadarpan** [comment]
I faced this issue yesterday while opening gatsby blog in my mobile chrome browser. The page finished loading, but it was blank.

After reloading, it loaded properly.

**Comment by btk** [comment]
It can't find a js file, that is expected to be there. Probably a serviceworker issue?

**Comment by jonniebigodes** [comment]
Working allright on my end. Tried it on chrome desktop(normal and incognito), firefox(normal and incognito) and edge. Same result, loads ok. That used to happen to me when visiting the site after gatsby v2 was implemented. And the site was migrated to that version. Had to reload 2 to 3 times to get the actual content. After a bit of detective work it looked like it was related to the service worker

**Comment by davidbailey00** [comment]
Hey there, I think this should be fixed once #9907 is merged!
